changelog.d/16135.misc

```diff
@@ -0,0 +1 @@
+Describe which rate limiter was hit in logs.
```

Comment:
Considered `.feature` but this only helps the odd admin occasionally, so I left it as `.misc`.